styles.css

```diff
@@ -0,0 +1,101 @@
+body {
+  margin: 0;
+  padding: 0;
+  flex-direction: column;
+  align-items: baseline;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #3c3a39;
+  color: white;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  padding-top: 17px;
+}
+
+.hamburger {
+  padding: 20px;
+}
+
+.name {
+  padding: 20px;
+}
+
+.headline {
+  display: flex;
+  font-family: 'Inter' Verdana, Geneva, Tahoma, sans-serif;
```

Comment:
- [ ] It seems that you have used different fonts compared to the Figma design for different sections of your website. Kindly ensure that you follow the design parameters provided by Figma exactly mainly because :
     - As web developers our first priority is to make what the employer wants and how they want it so this project can be considered good practice
     - This design has been made by professionals who have good idea of UI and UX so the design will look nice and it is proven to work,